server/helix/settings.py

```diff
@@ -244,7 +248,7 @@
 # Django storage
 
 # https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html
-if HELIX_ENVIRONMENT in ('production', 'nightly'):
+if HELIX_ENVIRONMENT in (PRODUCTION, NIGHTLY):
```

Comment:
```suggestion
if HELIX_ENVIRONMENT not in (DEVELOPMENT):
```
NOTE: ALPHA was missing in the list